get-started/composer-collections.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Organise Requests
```

Comment:
I think we prefer using US English instead of British English, so this should be Organi**z**e, with "Z"